ui/intermittents-view/helpers.js

```diff
@@ -0,0 +1,74 @@
+import moment from 'moment';
+
+//be sure to wrap date arg in a moment()
+export const ISODate = function formatISODate(date) {
+  return date.format('YYYY-MM-DD');
+};
+
+export const prettyDate = function formatPrettyDate(date) {
+  return moment(date).format('ddd MMM D, YYYY');
+};
+
+export const setDateRange = function setISODateRange(day, numDays) {
+  const to = ISODate(day);
+  const from = ISODate(day.subtract(numDays, 'days'));
+  return { from, to };
+};
+
+export const formatBugs = function formatBugsForBugzilla(data) {
+  let bugs = '';
+  for (let i = 0; i < data.length; i++) {
+    bugs += `${data[i].bug_id},`;
+  }
+  return bugs;
+};
+
+export const mergeData = function mergeDataFromTwoApis(data, bugs) {
+  let dict = {};
+  for (let i = 0; i < data.length; i++) {
+    dict[data[i].bug_id] = data[i].bug_count;
+  }
+
+  for (let i = 0; i < bugs.length; i++) {
+    let match = dict[bugs[i].id];
+    if (match) {
+      bugs[i].count = match;
+    }
+  }
+
+  bugs.sort(function (a, b) {
+    return b.count - a.count;
+  });
+  return bugs;
+};
+
+export const calculateMetrics = function calculateMetricsForGraphs(data) {
+  let dateCounts = [];
```

Comment:
These 3 arrays can actually be ``const`` since that doesn't limit adding entries, only reassigning the variable to a new object.

Reply:
Good point!